crypto/fipsmodule/ec/gfp_p256.c

```diff
@@ -70,18 +70,16 @@ void GFp_p256_scalar_sqr_rep_mont(ScalarMont r, const ScalarMont a, Limb rep) {
 
 #if !defined(OPENSSL_X86_64)
 
-#include <string.h>
-
 /* TODO(perf): Optimize these. */
 
 void GFp_nistz256_select_w5(P256_POINT *out, const P256_POINT table[16],
                             int index) {
   assert(index >= 0);
   size_t index_s = (size_t)index; /* XXX: constant time? */
 
-  alignas(32) Elem x; memset(x, 0, sizeof(x));
-  alignas(32) Elem y; memset(y, 0, sizeof(y));
-  alignas(32) Elem z; memset(z, 0, sizeof(z));
+  alignas(32) Elem x; limbs_zero(x, P256_LIMBS);
```

Comment:
Here and below, this can be written as
```c
alignas(32) Elem x = {0};
```

Reply:
Thanks. I made a TODO about this: issue #844.